cognite/client/_api/datapoints.py

```diff
@@ -1041,7 +1041,13 @@ def _get_datapoints(
             "ignoreUnknownIds": task.ignore_unknown_ids,
             "limit": min(window.limit, task.request_limit),
         }
-        res = self.client._post(self.client._RESOURCE_PATH + "/list", json=payload).json()["items"]
+
+        # Retry JSONDecodeError because we sometimes receive a truncated body from this endpoint.
+        # TODO: follow up with timelords and remove this retrying once the root cause has been found and fixed.
```

Comment:
maybe add a Jira task also to make sure this isn't forgotten

Reply:
Will do 👍